jobrunner/bin/update-jobrunner.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+ref=${1:-HEAD}
+
+cd /srv/jobrunner/code
+git fetch
+git checkout "$ref"
```

Comment:
this should be fine, assuming the production repository remains clean - other options include cribbing [what openprescribing does](https://github.com/ebmdatalab/openprescribing/blob/main/fabfile.py#L76) (`git checkout --force "origin/$ref"`)

Reply:
nice, will do